### PR TITLE
Reversing a string in different methods - 3 Implementations

### PR DESCRIPTION
**Pull request by Ritabrata1080**
- [x] File header is updated
- [x] No new warnings added by the PR
- [x] No commented code is present
- [x] Comments are added for any complex functionality

### CODE REVIEW

**Review by AGNIVA029 — DISMISSED** [review]

**Review by AGNIVA029 — DISMISSED** [review]
LGTM

File: src/strings/ReverseString.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            reverseStringUsingRecursion(s.substring(0, s.length() - 1));
+        }
+    }
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String str = sc.nextLine();
+        //System.out.println(reverseStringWithStack(str));
+        //System.out.println(reverseStringWithAnArray(str));
+        reverseStringUsingRecursion(str);
```
Comment (avanura23) [inline src/strings/ReverseString.java:63]: This implementation is using the method to print the reversal of the string. What if we have to use the reversed string further? Store it in a variable or print it in the main method.

Comment (Ritabrata1080) [reply src/strings/ReverseString.java]: Addressed this comment.

**Review by avanura23 — APPROVED** [review]
Good. Can be merged now.

**Pull request merged**